engine/battle_anims/anim_commands.asm

```diff
@@ -24,11 +24,11 @@ _PlayBattleAnim:
 	call BattleAnimRequestPals
 	call BattleAnimDelayFrame
 
-	ld c, 1
+	ld c, VBLANK_CUTSCENE
 	ldh a, [rKEY1]
 	bit 7, a ; check CGB double speed mode
 	jr nz, .got_speed
-	ld c, 3
+	ld c, VBLANK_CUTSCENE_CGB
 
 .got_speed
```

Comment:
This is not really a CGB check... Its a double speed check. So it wouldn't be accurate imo to call it VBLANK_CUTSCENE_CGB.

Furthermore, I think you got this backwards, unless i'm reading it wrong...

if bit 7 of rKEY1 is set (Double-speed mode), then we set `c` to `1`.

If bit 7 of rKEY1 is not set (Single Speed), then we set `c` to `3`.

Reply:
The thing is, vblank handler 3 was unused in DMG-supporting G/S, but Crystal repurposes it to call `ForceUpdateCGBPals` if `[hCGBPalUpdate]` is true. Without reference to which code happens to use that handler, I think "CGB cutscene" makes sense for it.

(Compare with `VBLANK_SERIAL`, which is only used by the printer and could be called `VBLANK_PRINTER`, but what actually distinguishes it is calling `AskSerial`.)

(Compare also with `VBLANK_CREDITS`, which IMO is specifically tuned to the credits -- e.g. only updating SCX but not SCY.)